### DISCUSSION TITLE
exchangelib.errors.UnknownTimeZone with Pyinstaller

### DISCUSSION THREAD

**Discussion by oONIo** [ecederstrand/exchangelib#921] [Q&A]
Dear All,

I wrote a small program that send email with attachment using exchangelib.
It is perfect to do the job.
Then, I was trying to pack it into a .exe file and share with my sister by using pyinstaller

The conversion finished successfully, but once I tried to run the Mail.exe, I have the following error:
  `..........`
  `File "exchangelib\fields.py", line 8, in <module>`
  `File "<frozen importlib._bootstrap>", line 1007, in _find_and_load`
  `File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked`
  `File "<frozen importlib._bootstrap>", line 680, in _load_unlocked`
  `File "PyInstaller\loader\pyimod03_importers.py", line 540, in exec_module`
  `File "exchangelib\ewsdatetime.py", line 321, in <module>`
  `File "exchangelib\ewsdatetime.py", line 216, in __new__`
`exchangelib.errors.UnknownTimeZone: No time zone found with key UTC`
`[18444] Failed to execute script Mail` 

Tried to Google but not much help on this issue.

Please kindly give me some advice to solve this.

Cheers

**Comment by ecederstrand** [comment]
This has been answered in https://github.com/ecederstrand/exchangelib/issues/897

**Reply by oONIo** [reply]
Thanks, I ve read the thread #897, and it seems having no solution to the issue. I am trying to find out a way that might include the missing package.

**Reply by ecederstrand** [reply]
The last message points to the workaround suggested in https://stackoverflow.com/questions/67711882/exchangelib-and-pyinstaller-zoneinfo-tzdata-utc-issue

**Reply by oONIo** [reply]
Thanks, I followed your suggestion.

I copied `tzdata` folder into the Project folder that under `/dist/`, e.g. /dist/Mail/tzdata.

Then there is no more error while I am running /dist/Mail/Mail.exe.

exchangelib is the greatest module with Office 365.
